### DISCUSSION TITLE
Speedtest tracker shows too little during download

### DISCUSSION THREAD

**Discussion by whazza1983** [alexjustesen/speedtest-tracker#1720] [Q&A] [answered]
Speedtest-Tracker
![Speedtracker](https://github.com/user-attachments/assets/514fd440-0593-4f8b-8973-38514c2e1228)

Speedtest Net Webseite
![speedtest_web](https://github.com/user-attachments/assets/116595b6-eb27-40d3-b3e0-6968f0fe5ca8)

Why does the speed test tracker show less?

**Comment by svenvg93** [comment]
The Speedtest track uses the Ookla cli client, which means the tests are using the same methods .

The only reason for a difference can be the Speedtest server used for tests or the medium tested with (wired / wireless).

**Comment by whazza1983** [comment]
I use the same servers for both. And I only use one device, my laptop, to test it.

Speedtest Tracker Server
61216 │ Server.camp │ Kassel │ Germany │ 1
![Speedtracker_server](https://github.com/user-attachments/assets/b15451a8-27b6-4648-8b37-e8c4544e2ed9)

and Speedtest Net Webseite

![speedtest_web_server](https://github.com/user-attachments/assets/d66935a5-a465-4073-b5a9-e4d336e900a4)

I tested in Docker network bridge and host

**Reply by whazza1983** [reply]
Now it works when I run the speed test myself and it shows the correct values. But when I start it via cron, the values ​​are not correct.
Self-started: 600.00 Mbps - 950.00 Mbps

By cron: 28.00 Mbps - 80.00 Mbps

**Reply by svenvg93** [reply]
Are you running the cron on the hour mark like 12:00, 13:00 etc. 
We have seen in the past that due to congestion on the network or speedtest server these results might be lower. The best way would be to test a couple of minutes later like 12:06, 13:06 etc.

**Reply by whazza1983** [reply]
I use wired. I have it running on my Raspberry Pi 5. As I said, when I start it myself the values ​​are correct. Only with SPEEDTEST_SCHEDULE, which I have set to every 2 hours (0 */2 * * *), the values ​​are no longer correct.

**Reply by svenvg93** [reply, marked as answer]
You might want to change the cron to like `6 */2 * * *` This will run the test 6 minutes after the second hour. 
We have seen in the past that due to congestion on the network or speedtest server these results might be lower when running on the hour mark.

**Reply by whazza1983** [reply]
I have now set it to your value. I will let you know in 6 hours whether it works now. 

And thank you for the replies

**Reply by svenvg93** [reply]
Your welcome! Let me know the outcome :)

**Reply by whazza1983** [reply]
It looks like it works. Great.. 
Thanks again
